torchaudio/transforms.py

```diff
@@ -471,3 +518,36 @@ def __call__(self, x_mu):
 
     def __repr__(self):
         return self.__class__.__name__ + '()'
+
+
+def dct(x, norm=None):
+    """
+    from https://github.com/zh217/torch-dct/blob/master/torch_dct/_dct.py
+    Discrete Cosine Transform, Type II (a.k.a. the DCT)
+    For the meaning of the parameter `norm`, see:
+    https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.fftpack.dct.html
+    :param x: the input signal
+    :param norm: the normalization, None or 'ortho'
+    :return: the DCT-II of the signal over the last dimension
+    """
+    x_shape = x.shape
+    N = x_shape[-1]
+    x = x.contiguous().view(-1, N)
```

Comment:
Also, dct should be in a transform class and not a standalone function.  It can be a method of the MFCC class if you like, but it doesn't make sense to have it as a separate function.

Reply:
Thanks, this looks good to me. I added it as a method of the MFCC class. I think eventually it makes sense to have it come from an outside "scipy.fftpack" torch equivalent but that doesn't really exist right now.